docs/installation_configuration_guide/DeploymentManagerConfiguration.md

```diff
@@ -76,6 +76,8 @@ spec:
           value: /opt/nussknacker/conf/application.conf,/data/modelConfig.conf
         - name: DEPLOYMENT_CONFIG_FILE
           value: /data/deploymentConfig.conf
+        - name: LOGBACK_FILE
+          value: /data/logback.xml
```

Comment:
its rather config than data but apparently we put data in root /data folder and not in /opt/nussknacker...